example/example/spiders/books_05_1.py

```diff
@@ -12,30 +12,30 @@
 """
 
 import scrapy
-
+from example.autoextract import ProductPage
```

Comment:
I wonder if we can make example imports relative, the way they are placed by black (?) look a bit weird otherwise. This applies to other files here as well.

Reply:
> Perhaps what you meant was that the linter (i.e. https://github.com/PyCQA/isort) moved this line from down below to up here?

sorry, that was exactly my concern 

> I think he meant from ...autoextract import ProductPage

yes, that's what I meant :) Sorry folks for an unclear message. It's not a big deal.